core/integration/features/verified/0011-MARA-006.feature

```diff
@@ -98,3 +97,298 @@ Feature: check margin account with partially filled order
+  
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader3 | ETH/DEC20 | 801         | 961    | 1201    | 1602    |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader3 | USD   | ETH/DEC20 | 0      | 90000   |
+
+    When the parties place the following orders with ticks:
+      | party   | market id | side | volume | price | resulting trades | type       | tif     | reference   |expires in |
+      | trader3 | ETH/DEC20 | buy  | 10     | 45    | 0                | TYPE_LIMIT | TIF_GTT | buy-order-5 |   3       |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader3 | ETH/DEC20 | 401         | 481    | 601     | 802    |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader3 | USD   | ETH/DEC20 | 601    | 89399   |
+
+    Then the network moves ahead "7" blocks
+    #GTT order expires
+     And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader3 | ETH/DEC20 | 401         | 481    | 601     | 802     |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader3 | USD   | ETH/DEC20 | 0      | 90000   |
+
+    # now we create a case when trader 4 place a GTC order first and then GTT order
+    When the parties place the following orders with ticks:
+      | party   | market id | side | volume | price | resulting trades | type       | tif     | reference   |expires in |
+      | trader4 | ETH/DEC20 | buy  | 5      | 45    | 0                | TYPE_LIMIT | TIF_GTC | buy-order-5 |           |
+      | trader4 | ETH/DEC20 | buy  | 10     | 45    | 0                | TYPE_LIMIT | TIF_GTT | buy-order-5 |     3     |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader4 | ETH/DEC20 | 601         | 721    | 901     | 1202    |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader4 | USD   | ETH/DEC20 | 901    | 89099   |
+
+    Then the network moves ahead "7" blocks
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader4 | ETH/DEC20 | 601         | 721    | 901     | 1202    |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader4 | USD   | ETH/DEC20 | 901    | 89099   |
+
+  Scenario: 003 check margin for GFN order type 0011-MARA-009
```

Comment:
Wouldn't it make sense to trigger an auction and check that the order gets cancelled and margin level is updated? Not much to test in continuous trading I think.